README.md

```diff
@@ -121,6 +120,12 @@ url | proxy-server.subman.example.com:3128
 username | proxyuser
 password | password
 
+Development of the Subscription-Manager Deployment Ansible role
+---------------------------------------------------------------
+Edit the `vagrant/requirements.yml` file and change the src property to 
+`- src: git+file:///your/development/path/to/ansible-role-subman-devel`. 
+This will pull the latest commit from this path and use it for deployment. 
```

Comment:
Can you please add here link to the remote repository? This would make no sense to newcomers.

Reply:
Sorry for confusing. The `/your/development/path/to/ansible-role-subman-devel` is clone of remote repository from here: https://github.com/candlepin/ansible-role-subman-devel, right? I would just add here link to this project. Thoughts?

Reply:
That is a really good idea.